packages/integration-tests/src/entities/User.test.ts

```diff
@@ -1,11 +1,55 @@
 import { User } from "@src/entities";
+import { insertUser } from "@src/entities/inserts";
+import { PasswordValue } from "@src/entities/types";
 import { newEntityManager } from "@src/setupDbTests";
 
+const PASSWORD = "correct.horse.battery.staple";
+const PASSWORD_ENCODED = "Y29ycmVjdC5ob3JzZS5iYXR0ZXJ5LnN0YXBsZQ==";
+
 describe("User", () => {
   it("custom type is exposed", async () => {
     const em = newEntityManager();
     // @ts-expect-error
     const a1 = em.create(User, { name: "a1", email: "test@test.com", ipAddress: "127.0.0.1" });
     await expect(em.flush()).resolves.toEqual([a1]);
   });
+
+  it("can interact with serde value", async () => {
+    const em = newEntityManager();
+    const a1 = em.create(User, {
+      name: "a1",
+      email: "test@test.com",
+      password: PasswordValue.fromPlainText(PASSWORD),
+    });
+    await expect(em.flush()).resolves.toEqual([a1]);
+    await expect(a1.password?.encoded).toEqual(PASSWORD_ENCODED);
```

Comment:
These assertions look good, could add another, using our `select` helper to do like:

```
expect((await select("authors"))[0].password).toEqual(PASSWORD_ENCODED);
```

Just to make sure the encoded value goes all the way through to the db.